### PR TITLE
Additional command aliases

### PR DESCRIPTION
**Pull request by P0keDev**
Adds a handful of aliases to the lootrun command, as well as the Wynncraft guild and party commands.

**Lootrun**:
/lootrun -> /lr or /loot
/lr load -> /lr l
/lr save -> /lr s
/lr delete -> /lr d
/lr record -> /lr r
/lr clear -> /lr c

**Party**:
/party join -> /party j
/party invite -> /party i
/party leave -> /party l

**Guild**:
/guild attack -> /guild att/a
/guild defend -> /guild def/d
/guild contribute -> /guild c

Suggested by Goden and Seasalt.

### CODE REVIEW

**Comment by McPlayHD** [comment]
I'm not happy with that as it is right now...
Can you please add a way that the commands get sent to the server as long as you are not on the Wynncraft server?

File: src/main/java/com/wynntils/modules/chat/events/ClientEvents.java [hunk 1]
```diff
@@ -62,11 +62,22 @@ public void onChatReceived(ClientChatReceivedEvent e) {
...
+     *        l -> leave
+     *        c -> create
      */
     @SubscribeEvent
     public void commandReplacements(ClientChatEvent e) {
         if (e.getMessage().startsWith("/tell")) e.setMessage(e.getMessage().replaceFirst("/tell", "/msg"));
         else if (e.getMessage().startsWith("/xp")) e.setMessage(e.getMessage().replaceFirst("/xp", "/guild xp"));
+        else if (e.getMessage().startsWith("/gu")) e.setMessage(e.getMessage().replaceFirst("att$", "attack").replaceFirst("a$", "attack").replaceFirst("def$", "defend").replaceFirst("d$", "defend").replaceFirst("c$", "contribute"));
+        else if (e.getMessage().startsWith("/pa")) e.setMessage(e.getMessage().replaceFirst(" j ", " join ").replaceFirst(" i ", " invite ").replaceFirst("l$", "leave").replaceFirst("c$", "create"));
+        System.out.println(e.getMessage());
```
Comment (HighCrit) [inline src/main/java/com/wynntils/modules/chat/events/ClientEvents.java]: Remove debug line

File: src/main/java/com/wynntils/modules/chat/events/ClientEvents.java [hunk 2]
```diff
@@ -62,11 +62,22 @@ public void onChatReceived(ClientChatReceivedEvent e) {
...
+     *        def/d -> defend
+     *        c -> contribute
+     * /party j -> join
+     *        i -> invite
+     *        l -> leave
+     *        c -> create
      */
     @SubscribeEvent
     public void commandReplacements(ClientChatEvent e) {
         if (e.getMessage().startsWith("/tell")) e.setMessage(e.getMessage().replaceFirst("/tell", "/msg"));
```
Comment (HighCrit) [inline src/main/java/com/wynntils/modules/chat/events/ClientEvents.java:76]: Send message as is when not on Wynncraft

**Review by HighCrit — CHANGES_REQUESTED** [review]

File: src/main/java/com/wynntils/modules/chat/events/ClientEvents.java (hunk 2 above)
Comment (coehlrich) [reply src/main/java/com/wynntils/modules/chat/events/ClientEvents.java:76]: That class already only receives events when on Wynncraft

File: src/main/java/com/wynntils/modules/chat/events/ClientEvents.java (hunk 1 above)
Comment (P0keDev) [reply src/main/java/com/wynntils/modules/chat/events/ClientEvents.java]: whoops, fixed

**Pull request merged**